### DISCUSSION TITLE
Extensions module

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#11581] [Feature Requests]
A module that lists all the extensions (including custom api and webhooks) with switches to enable/disable them would be neat little usability feature.

Are there any plans to allow uploading extensions directly via the App by the time the marketplace is live?

**Comment by benhaynes** [comment]
> Are there any plans to allow uploading extensions directly via the App by the time the marketplace is live?

Yup! There's a discussion for it here somewhere, and we already have rough designs ready. Ideally getting this done within a few months.

**Reply by hermesalvesbr** [reply]
Few months? :(

I am very much looking forward to this marketplace.

**Reply by benhaynes** [reply]
It takes time to do things right. 🙂

**Reply by hermesalvesbr** [reply]
It's true.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
